Client/core/CGraphStats.cpp

```diff
@@ -200,11 +203,12 @@ void CGraphStats::Draw()
 
     CGraphicsInterface* pGraphics = g_pCore->GetGraphics();
 
+    uint  uiViewportWidth = pGraphics->GetViewportWidth();
     uint  uiViewportHeight = pGraphics->GetViewportHeight();
     uint  uiOriginX = 10;
-    uint  uiOriginY = std::min<int>(500, uiViewportHeight - 10);
-    uint  uiSizeX = GRAPHSTAT_HISTORY_SIZE;
-    uint  uiSizeY = 150;
+    uint  uiOriginY = uiViewportHeight / 2;
+    uint  uiSizeX = uiViewportWidth / 4;
+    uint  uiSizeY = uiViewportHeight / 4;
```

Comment:
Why these values?

Reply:
Because the graph was always alligned to left side right under the chat window.